hubmap/hubmap/settings.py

```diff
@@ -76,11 +107,29 @@
                 'django.template.context_processors.debug',
                 'django.template.context_processors.request',
                 'django.contrib.auth.context_processors.auth',
+                'social_django.context_processors.backends',
+                'social_django.context_processors.login_redirect',
                 'django.contrib.messages.context_processors.messages',
             ],
         },
     },
 ]
+# SECURITY WARNING: keep the secret key used in production secret!
+# SECRET_KEY = 'g+^rn4oz4$##*i6i=z)!vgwufp)4#q(fx5i*1m_8=7s88=rtdr'
+# Authentication, including Globus SSO
+AUTHENTICATION_BACKENDS = [
+    'social_core.backends.globus.GlobusOpenIdConnect',
+    'django.contrib.auth.backends.ModelBackend',
+    'rest_framework_social_oauth2.backends.DjangoOAuth2',
+]
+
+# for giygas.compbio.cs.cmu.edu, change in the future
```

Comment:
Are these keys/secrets for my desktop machine?

Reply:
No, they are not.